cli/dcoscli/package/main.py

```diff
@@ -356,35 +332,31 @@ def _install(package_name, package_version, options_path, app_id, cli, app,
             emitter.publish('Exiting installation.')
             return 0
 
-    options = pkg.options(pkg_revision, user_options)
+    # render options before start installation
+    options = pkg.options(user_options)
 
-    revision_map = pkg.package_revisions_map()
-    package_version = revision_map.get(pkg_revision)
+    if app and pkg.has_mustache_definition():
```

Comment:
I think we should change what Cosmos's describe returns so that it doesn't include the mustache. This means that we should just tell Cosmos to install it. If it doesn't exists then Cosmos should just return a well known error and we handle it on our side.